packages/geo/src/lib/map/shared/mapLayer.directive.ts

```diff
@@ -0,0 +1,56 @@
+import { Directive, AfterViewInit, OnDestroy, Self, OnInit, EventEmitter, Output } from '@angular/core';
+import { IgoMap } from './map';
+import { MapBrowserComponent } from '../map-browser/map-browser.component';
+import { NetworkService, ConnectionState } from '@igo2/core';
+import { Subscription } from 'rxjs';
+import { Layer } from '../../layer/shared/layers/layer';
+
+@Directive({
+    selector: '[igoMapLayer]'
+  })
+export class MapLayerDirective implements AfterViewInit {
```

Comment:
The name of the class is not significant

Reply:
is igoChangeMapSourceUrl would be ok?

Reply:
MapLayerOfflineDirective ?
MapOfflineDirective ?